### ISSUE TITLE
Clementine writes weird files to the home directory when switching tracks

### ISSUE THREAD

**Issue by Clementine-Issue-Importer** [clementine-player/Clementine#1782]
_From [erissiva](https://code.google.com/u/117383083083809959915/) on April 18, 2011 21:28:10_

What steps will reproduce the problem? 1. Start Clementine
2. Start playback of any track
3. Switch to "next" or "previous" track What is the expected output? What do you see instead? Expected output is to have it switch to the next track without writing any weird files to my home directory. Instead, Clementine insists on writing weird files with foreign characters to my home folder. What version of the product are you using? On what operating system? Clementine is 0.7.1 from the PPA.
OS is Ubuntu 11.04 Beta 2 (Natty).

Screenshot is attached.

Basically it just writes a number of files with stings of broken characters to ~/.  If I do not delete the files, they remain unchanged. If I delete them, as soon as Clementine switches tracks it writes them again. I've checked my settings and cannot, for the life of me, figure out what would be causing this.

**Attachment:** [Selection_006.png](http://code.google.com/p/clementine-player/issues/detail?id=1782)

_Original issue: http://code.google.com/p/clementine-player/issues/detail?id=1782_
